config/selectionCfg_ETau_UL16.cfg

```diff
@@ -0,0 +1,1279 @@
+
+btagM = (bjet1_bID_deepFlavor > 0.2489 && bjet2_bID_deepFlavor < 0.2489) || (bjet1_bID_deepFlavor < 0.2489 && bjet2_bID_deepFlavor > 0.2489)    #only 1 jet with medium btag working point
+btagLL = bjet1_bID_deepFlavor > 0.0480 && bjet2_bID_deepFlavor > 0.0480 #both jets with loose btag working point
+btagMM = bjet1_bID_deepFlavor > 0.2489 && bjet2_bID_deepFlavor > 0.2489 #both jets with medium btag working point
+
+# H-bbtag requirements - ParticleNet  
+pNetBTagH = fatjet_particleNetMDJetTags_score >= 0.9883
+pNetBTagM = fatjet_particleNetMDJetTags_score >= 0.9735
+pNetBTagL = fatjet_particleNetMDJetTags_score >= 0.9137
+
+massCut    = bH_mass > 50 && bH_mass < 270 && tauH_mass > 20 && tauH_mass < 130
+massCutInv = (bH_mass < 50 || bH_mass > 270) && (tauH_mass < 20 || tauH_mass > 130)
+
+# ABCD regions used in the analysis - DeepTau
+SR       = isOS != 0 && dau1_eleMVAiso == 1 && dau2_deepTauVsJet >= 5 # signal region: opposite sign, iso taus
+SStight  = isOS == 0 && dau1_eleMVAiso == 1 && dau2_deepTauVsJet >= 5                          # B region
+OSinviso = isOS != 0 && dau1_eleMVAiso == 1 && dau2_deepTauVsJet >= 1 && dau2_deepTauVsJet < 5 # C region
+SSinviso = isOS == 0 && dau1_eleMVAiso == 1 && dau2_deepTauVsJet >= 1 && dau2_deepTauVsJet < 5 # D region
+
+baselineMcut    = baseline, massCut
+baselineInvMcut = baseline, massCutInv
+
+res1b = baseline, btagM , isBoosted != 1, massCut
+res2b = baseline, btagMM, isBoosted != 1, massCut
+
+boostedL_pnet = baseline_boosted, pNetBTagL
+boostedM_pnet = baseline_boosted, pNetBTagM
+boosted_semi  = !(pairType == 1 && nleps == 0 && (isLeptrigger || (isMETtriggerNoThresh and metnomu_et > 160)) && isBoosted == 1 && fatjet_particleNetMDJetTags_score >= 0.9137), baseline, btagLL, isBoosted==1, fatjet_hasMatchedSj==1, massCut
+boosted = boostedL_pnet || boosted_semi
+
+res1bmHH250_335   = res1b, mHH1
+res1bmHH335_475   = res1b, mHH2
+res1bmHH475_725   = res1b, mHH3
+res1bmHH725_1100  = res1b, mHH4
+res1bmHH1100_3500 = res1b, mHH5
+
+res1bmHH250_335   = res1b, mHH1
+res1bmHH335_475   = res1b, mHH2
+res1bmHH475_725   = res1b, mHH3
+res1bmHH725_1100  = res1b, mHH4
+res1bmHH1100_3500 = res1b, mHH5
+
+boostedmHH250_625	= boostedL_pnet, mHH1b
+boostedmHH625_775	= boostedL_pnet, mHH2b
+boostedmHH775_1100	= boostedL_pnet, mHH3b
+boostedmHH1100_3500 = boostedL_pnet, mHH4b
+
+#########################################################################
+#########################################################################
+
+## weights to be applied for a certain selections when calling Fill()
+## multiple weights are passed as list and are multiplied together
+## NOTE: no weight is applied for data (the simple Fill() is used)
+[selectionWeights]
+baseline		 = MC_weight, PUReweight, L1pref_weight, trigSF, IdFakeSF_deep_2d, PUjetID_SF, bTagweightReshape
+baseline_boosted = MC_weight, PUReweight, L1pref_weight, trigSF, IdFakeSF_deep_2d, PUjetID_SF, bTagweightReshape
+
```

Comment:
I do not know if bTagweightReshape has to be applied here.

Reply:
We decided to remove it from the boosted category (SF=1).